src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/events/BitbucketCloudPullRequestEvent.java

```diff
@@ -73,7 +73,7 @@ private void reconstructMissingData() {
                     if (this.pullRequest.getSource().getRepository().getOwnerName().equals(this.pullRequest.getAuthorLogin())) {
```

Comment:
Whatever this code was doing, it can no longer work. For example, `.source.repository.full_name` might be `jglick/somerepo`, but the `jglick` part is no longer accessible from the API.

Reply:
Investigating failures in `BitbucketCloudPullRequestEventTest`. There seems to be a real problem here—not a regression from my patch, but brought to light by the updates to mock API responses.

Reply:
(Manually triggered scans and pull request builds seem to work fine as is, but IIUC this code is dealing specifically with webhooks.)

Reply:
#37 FTR